tests/framework/CodeInterfaceTests/PARCS/test_GA-PARCS-LP.xml

```diff
@@ -0,0 +1,344 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>test_loaddingpattern_GA-PARCS</name>
+    <author>khang</author>
+    <created>2022-07-12</created>
+    <classesTested>Models.Code.PARCS-GA</classesTested>
+    <description>
+       An example of using the the GA and PARCS to generate the optimized loadding pattern
+    </description>
+    <revisions>
+      <revision author="khang" date="2020-11-2">Test created</revision>
+    </revisions>
+  </TestInfo>
+  <RunInfo>
+    <WorkingDir>PARCS-RAVEN-GA</WorkingDir>
+    <Sequence>sampleGA-PARCS, print</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Files>
+    <Input name="parcs_input" type="parcsdata">parcs-input-gen.xml</Input>
+    <Input name="parcsperturb_input" type="perturb">coremap.xml</Input>
+    <Input name="input" type="input">input.inp</Input>
+  </Files>
+
+  <Models>
+    <Code name="MyParcs" subType="PARCS">
+      <executable>/cm/shared/apps/ncsu/TRACE_PARCS/PARCS-v32m19co_src/run/parcs_ifr.x</executable>
+      <sequence>parcs</sequence>
+    </Code>
+  </Models>
+  <Functions>
+    <External file="./constraints.py" name="impConstr1">
+      <variables>PinPowerPeaking,MaxFDH,max_boron</variables>
+    </External>
+    <External file="./constraints.py" name="impConstr2">
+      <variables>PinPowerPeaking,MaxFDH,max_boron</variables>
+    </External>
+    <External file="./constraints.py" name="impConstr3">
+      <variables>PinPowerPeaking,MaxFDH,max_boron</variables>
+    </External>
+  </Functions>
+  <Distributions>
+    <UniformDiscrete name='FA_dist'>
+      <lowerBound>0</lowerBound>
+      <upperBound>4</upperBound>
+      <strategy>withReplacement</strategy>
+    </UniformDiscrete>
+  </Distributions>
+  
+  <Optimizers>
+    <GeneticAlgorithm name="GAopt">
+      <samplerInit>
+       <limit>5</limit>
+        <initialSeed>42</initialSeed>
+        <writeSteps>every</writeSteps>
+        <type>max</type>
+      </samplerInit>
+
+      <GAparams>
+        <populationSize>5</populationSize>

```

Comment:
I would suggest to reduce the populationSize to 3. This is mainly because a lot of test files are added which are redundant since RAVEN can only test the mechanism.